fetch.bs

```diff
@@ -6710,6 +6748,362 @@ agent's <a>CORS-preflight cache</a> for which there is a <a>cache entry match</a
+<dfn for=PermissionPolicy enum-value>"deferred-fetch"</dfn>. Its
+<a for="policy-controlled feature">default allowlist</a> is "<code>self</code>".
+
+<p>This specification defined a <a>policy-controlled feature</a> identified by the string
+<dfn for=PermissionPolicy enum-value>"deferred-fetch-minimal"</dfn>. Its
+<a for="policy-controlled feature">default allowlist</a> is "<code>*</code>".
+
+<p>The <dfn>optional subframe deferred-fetch quota</dfn> is 64 kibibytes.
+<p>The <dfn>minimal subframe deferred-fetch quota</dfn> is 8 kibibytes.
+<p>The <dfn>deferred-fetch delegated quota</dfn> is 128 kibibytes.
+
+<div algorithm>
+<p>Two {{Document}}s <var>a</var> and <var>b</var> are said to
+<dfn>share deferred-fetch quota</dfn> if <var>a</var>'s
+<a>relevant agent</a> is <var>b</var>'s <a>relevant agent</a>, and <var>a</var>'s
+<a for=Document>URL</a>'s <a for=url>origin</a> is <a>same origin</a> with <var>b</var>'s
+<a for=Document>URL</a>'s <a for=url>origin</a>.
+</div>
+
+
+<div algorithm>
+<p>To get the <dfn>available deferred-fetch quota</dfn> given a {{Document}}
+<var>document</var> and an <a for=/>origin</a>-or-null <var>origin</var>:
+
+<ol>
+ <li><p>Let <var>quota</var> be zero.
+
+ <li>
+  <p>Let <var>quotaForOrigin</var> be 64 kibibytes.
+  <p class=note>
+
+ <li>
+  <p>For each <var>otherNavigable</var> in <var>document</var>'s <a>node navigable</a>'s
+  <a for=navigable>top-level traversable</a>'s <a>inclusive descendant navigables</a>:
+
+  <p class=note>This algorithm iterates over the entire navigable tree. It accumulates quota from
+  the <a for=/>top-level traversable</a>, and from subframes who inherit quota from cross-origin or
+  cross-agent navigables. Subsequently, it subtracts the quota delegated to cross-origin or
+  cross-agent subframes, as well as quota spent on pending deferred fetch requests.
+
+  <ol>
+   <li><p>Let <var>otherDocument</var> be <var>otherNavigable</var>'s <a>active document</a>.
+
+   <li>
+    <p>If <var>document</var> and <var>otherDocument</var> <a>share deferred-fetch quota</a>,
+    then:
+    <ol>
+     <li>
+      <p>If <var>otherDocument</var>'s <a>node navigable</a> is a
+      <a for=/>top-level traversable</a>, then:
+
+      <p class=note>Initialize the quota from the top-level traversable.
+
+      <ol>
+       <li><p>Assert: <var>quota</var> is zero.
+
+       <li><p>If <var>otherDocument</var> is not
+       <a>allowed to use</a> the <a>policy-controlled feature</a>
+       {{PermissionPolicy/"deferred-fetch"}}, then return 0.
+
+       <li>
+        <p>Set <var>quota</var> be 640 kibibytes.
+        <p class="note allow-2119">640kb should be enough for everyone.
+
+       <li><p>If <var>otherDocument</var> is <a>allowed to use</a> the
+       <a>policy-controlled feature</a> {{PermissionPolicy/"deferred-fetch-minimal"}}, then
+       decrement <var>quota</var> by <a>deferred-fetch delegated quota</a>.
+      </ol>
+
+     <li>
+      <p>Otherwise,
```

Comment:
In the rendered spec page, this `Otherwise`'s indentation is slightly misaligned with the above `If otherDocument’s node navigable is a top-level traversable, then:`

Reply:
Fixed